pkg/reconciler/revision/resources/queue_test.go

```diff
@@ -382,6 +382,39 @@ func TestMakeQueueContainer(t *testing.T) {
 				"SERVING_REQUEST_METRICS_BACKEND": "prometheus",
 			}),
 		},
+	}, {
+		name: "enable profiling",
+		rev: &v1alpha1.Revision{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: "foo",
+				Name:      "bar",
+				UID:       "1234",
+			},
+			Spec: v1alpha1.RevisionSpec{
+				RevisionSpec: v1beta1.RevisionSpec{
+					ContainerConcurrency: ptr.Int64(0),
+					TimeoutSeconds:       ptr.Int64(45),
+				},
+			},
+		},
+		lc: &logging.Config{},
+		tc: &tracingconfig.Config{},
+		oc: &metrics.ObservabilityConfig{EnableProfiling: true},
```

Comment:
Can we add test case when `EnableProfiling: false` ?

Reply:
This is tested via the other test cases through the default value defined here: https://github.com/knative/serving/pull/5277/files#diff-163887f1327a36aa0ffe49bbb82e5d1aR1131